<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddChargeDialog</class>
 <widget class="QDialog" name="AddChargeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>474</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Palatino Linotype</family>
    <pointsize>11</pointsize>
    <weight>9</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Add Charge</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 49, 91);
font: 75 11pt &quot;Palatino Linotype&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,2,1">
   <item row="2" column="0">
    <widget class="QFrame" name="frame">
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QPushButton" name="cancel_Button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(160, 160, 160);
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="add_charge_Button">
        <property name="font">
         <font>
          <family>Palatino Linotype</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(62, 146, 255);
font: 75 11pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Add Charge</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="case_information_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>5000</width>
       <height>5000</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);
font-weight: bold;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="case_number_label">
        <property name="font">
         <font>
          <family>Palatino Linotype</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="defendant_name_label">
        <property name="font">
         <font>
          <family>Palatino Linotype</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);
font-weight: bold;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,2,0">
      <item row="3" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Degree:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Offense:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Statute:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="offense_choice_box">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="statute_choice_box">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="degree_choice_box">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <item>
         <property name="text">
          <string>MM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UCM</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="clear_fields_Button">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(160, 160, 160);</string>
        </property>
        <property name="text">
         <string>Clear Fields</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="freeform_entry_checkBox">
        <property name="font">
         <font>
          <family>Palatino Linotype</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Enable freeform entry for offense/statute</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>statute_choice_box</tabstop>
  <tabstop>offense_choice_box</tabstop>
  <tabstop>add_charge_Button</tabstop>
  <tabstop>cancel_Button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>proceed_to_ability_to_pay()</slot>
  <slot>close_window()</slot>
  <slot>add_offense()</slot>
  <slot>amend_offense()</slot>
  <slot>proceed_to_sentencing()</slot>
 </slots>
</ui>
